kronolith/lib/Event.php

```diff
@@ -2855,6 +2890,9 @@ public function readForm(Kronolith_Event $existing = null)
                 if (!isset($newattendees[$email])) {
                     unset($attendees[$email]);
                 }
+                if (Kronolith::isUserEmail($this->creator, $email)) {
+                    $attendees[$email]['response'] = Horde_Util::getFormData('attendance');
+                }
             }
         }
```

Comment:
I think the text representation is the right way to go. We send itip responds to the organizer who in turn sends out updates (or doesnt). We should make it clear to the invited user that he does not actually "own" the event. We should never send out itip updates for an event we do not own.